packages/tabs/addon/components/nucleus-tabs/tab-list-item.js

```diff
@@ -0,0 +1,267 @@
+import Component from '@ember/component';
+import { classNames, attributeBindings, classNameBindings, tagName, layout as templateLayout } from '@ember-decorators/component';
+import layout from '../../templates/components/nucleus-tabs/tab-list-item';
+import { get, computed } from '@ember/object';
+import defaultProp from '@freshworks/core/utils/default-decorator';
+import { once } from '@ember/runloop';
+import { TABS_KEY_CODE } from '../../constants/nucleus-tabs'
+
+/**
+  __Usage:__
+
+  [Refer component page](/docs/components/nucleus-tabs)
+
+  @class Nucleus Tab List Item
+  @namespace Components
+  @extends Ember.Component
+  @private
+*/
+@tagName('button')
```

Comment:
Please use `{{nucleus-button}}`

Reply:
Tab and Button have completely different aria attributes, roles. Styling and behaviour also differs. Hence just changed the base tag as per our discussion to avoid any confusions.